scripts/build-docs.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-env node */
+'use strict';
+
+const path = require('path');
+const fsThen = require('fs-then-native');
+const mkdirp = require('mkdirp');
+
+const DEST_DIR = 'dist';
+const GITHUB_FORMAT = 'github';
+const KRAMDOWN_FORMAT = 'kramdown';
+const PC_DEV_CENTER_FORMAT = 'purecloudDevCenter';
+const SUPPORTED_DOC_OUTPUT_FORMATS = [GITHUB_FORMAT, KRAMDOWN_FORMAT, PC_DEV_CENTER_FORMAT];
+let docMdOutputFormat = process.env.DOC_MD_OUTPUT_FORMAT || PC_DEV_CENTER_FORMAT;
+if (SUPPORTED_DOC_OUTPUT_FORMATS.indexOf(docMdOutputFormat) < 0) {
+    console.error(`Unknown MD Output Format Specified: '${docMdOutputFormat}'`);
+    process.exit(1);
+}
+
+let relativeLinkExtension = '.md';
```

Comment:
nit, this should be linkExtension since it's global in this context.

Reply:
Yeah, the "relative" here applied to the fact that they were relative links. But then I see that I named it `linkExtension` in the helper 🙃